metaphor/snowflake/extractor.py

```diff
@@ -33,52 +27,45 @@
     SourceInfo,
     SQLSchema,
 )
+from metaphor.snowflake.auth import connect
```

Comment:
```suggestion
from metaphor.snowflake import auth
```
Align with other snowflake connectors?

Reply:
Good point. The reason why I import `auth` instead of `connect` is so that it can be monkey-patched in tests. We didn't have any tests for `SnowflakeExtractor` directly so it didn't matter.